src/tests

```diff
@@ -40,3 +40,6 @@ p({"class":"foam.nanos.test.Test","id":"e2e-json-serialization-test","descriptio
 
 p({"class":"foam.nanos.test.Test","id":"e2e-json-serialization-test","passed":2,"lastRun":"2019-04-16T21:34:50.064Z","lastDuration":87,"status":2,"output":"JS Serialization: {\"class\":\"foam.test.AllSerializableProperties\",\"intProp\":12,\"stringProp\":\"asdf\",\"fObjectArrayProp\":[{\"description\":\"An object in an array!\"},{\"description\":\"Another object in an array!\"}],\"objectProp\":[1,{\"class\":\"__Timestamp__\",\"value\":1555450490010}],\"fObjectPropertyProp\":{\"description\":\"some object\"},\"emailProp\":\"test@example.com\",\"enumProp\":1,\"dateProp\":819188640000,\"dateTimeProp\":819278624000,\"floatProp\":1.2345,\"longProp\":12341234,\"currencyProp\":342342,\"colorProp\":\"rgba(0, 0, 255, 0)\",\"mapProp\":{},\"imageProp\":\"/favicon/favicon-32x32.png\",\"urlProp\":\"https://google.com/\",\"passwordProp\":\"superSecret111!\",\"phoneNumberProp\":\"555-3455\"}\nSUCCESS: After JS Serialization object is still equivalent.\nSUCCESS: After Java Serialization object is still equivalent.\n"})
 p({"class":"foam.nanos.test.Test","id":"e2e-json-serialization-test","status":1})
+p({"class":"foam.nanos.test.Test","id":"e2e-json-serialization-test","code":"var jsonParser = foam.json.Parser.create({ strict: true });\n\nvar initialObject = foam.test.AllSerializableProperties.createPopulated();\n\nvar jsSerialized = foam.json.Network.stringify(initialObject);\n\nlog(\"JS Serialization: \", jsSerialized);\n\nvar afterJSSerialization = jsonParser.parseString(jsSerialized);\n\ntest(foam.util.equals(initialObject, afterJSSerialization), \"After JS Serialization object is still equivalent.\");\n\nx[\"serializationTestEchoService\"].echo(afterJSSerialization).then(function(afterJavaSerialization) {\n  var diff = initialObject.diff(afterJavaSerialization);\n  for ( var keys = Object.keys(diff), i = 0, key ; key = keys[i++] ; ) {\n    log(\"Differs on property \", key, \": \", diff[key]);\n  }\n  test(foam.util.equals(initialObject, afterJavaSerialization), \"After Java Serialization object is still equivalent.\");\n});\n"})
```

Comment:
I think we can cleanup this journal and just replace the line 39 with your updated line 43.